kong/plugins/prometheus/handler.lua

```diff
@@ -22,6 +22,12 @@ function PrometheusHandler.log(self, conf)
   if conf.per_consumer and message.consumer ~= nil then
     serialized.consumer = message.consumer.username
   end
+  if not conf.expose_services_tags and message.service ~= nil then
+    message.service.tags = {}
```

Comment:
maybe just `message.service.tags = nil` since you already test if it's a table later
so this function creates less garbage tables.